packages/components/addon/components/hds/pagination/nav/index.js

```diff
@@ -75,6 +83,46 @@ export default class HdsPaginationNavIndexComponent extends Component {
     return this.currentPage === this.totalPages;
   }
 
+  elliptize({ pages, current }) {
+    const array = Array.from(pages);
```

Comment:
you don't need to create a copy of the array, you can use directly `pages` I think (as far as I remember, arrays are not passed as reference)